src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -162,11 +166,10 @@ class JesBackend extends Backend with LazyLogging {
     val jesParameters = standardParameters(callGcsPath) ++ jesInputs ++ unsafeJesOutputs
 
     // Not possible to currently get stdout/stderr so redirect everything and hope the WDL isn't doing that too
-    val redirectedCommand = s"$commandLine > $LocalStdoutValue  2> $LocalStderrValue"
+    val redirectedCommand = s"$instantiatedCommandLine > $LocalStdoutValue 2> $LocalStderrValue"
```

Comment:
I don't see how script-ifying it resolves not being able to automagically capture stdout/stderr?

Either way i have the same issue w/ doing that that I did w/ their suggestion in the other thread, it's a moving part over a network and one which doesn't need to happen. If it's something we need to do so be it but otherwise it's a potential for error. Also the stdout/stderr shenanigans are temporary, or so they say ...